### PR TITLE
Updated manual synchornization example scene

### PR DESCRIPTION
**Pull request by lauri-rustanius**
**Description of changes**

Updated the manual map localization to be done in Unity coordinate space.

Relates to https://github.com/mapbox/unity/issues/547

**Reviewers**

@atripathi-mb @jim-martin

### CODE REVIEW

File: sdkproject/Assets/MapboxAR/Scripts/FollowTargetTransform.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
```
Comment (atripathi-mb) [inline sdkproject/Assets/MapboxAR/Scripts/FollowTargetTransform.cs]: Add namespace

File: sdkproject/Assets/MapboxAR/Scripts/RelocateMapByGPS.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
```
Comment (atripathi-mb) [inline sdkproject/Assets/MapboxAR/Scripts/RelocateMapByGPS.cs]: Add namespace

File: sdkproject/Assets/MapboxAR/Scripts/RotateOnYTargetTransform.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
```
Comment (atripathi-mb) [inline sdkproject/Assets/MapboxAR/Scripts/RotateOnYTargetTransform.cs]: Add namespace

File: sdkproject/Assets/MapboxAR/Scripts/UpdateMapPosByARPlaneY.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections;
```
Comment (atripathi-mb) [inline sdkproject/Assets/MapboxAR/Scripts/UpdateMapPosByARPlaneY.cs]: Add namespace.

File: sdkproject/ProjectSettings/ProjectSettings.asset [hunk 5]
```diff
@@ -12,7 +12,7 @@ PlayerSettings:
   useOnDemandResources: 0
   accelerometerFrequency: 60
   companyName: Mapbox
-  productName: MBX SDK
```
Comment (atripathi-mb) [inline sdkproject/ProjectSettings/ProjectSettings.asset]: Revert back to MBX SDK

File: sdkproject/ProjectSettings/ProjectSettings.asset [hunk 6]
```diff
@@ -161,7 +161,7 @@ PlayerSettings:
     Android: com.mapbox.unitysdk
     Standalone: unity.Mapbox.MBX SDK
     Tizen: com.mapbox.unitysdk
-    iOS: com.mapbox.MBX-SDK
+    iOS: com.mapbox.manual
```
Comment (atripathi-mb) [inline sdkproject/ProjectSettings/ProjectSettings.asset]: Revert back to MBX SDK

**Comment by lauri-rustanius** [comment]
@atripathi-mb added namespaces.

**Pull request merged**